src/components/WorkOrder/Image/Camera.js

```diff
@@ -82,6 +82,7 @@ export class Camera {
   stop() {
     this.webcamElement.srcObject.getTracks().map(function(val) {
       val.stop();
+      return true;
```

Comment:
🙌 I kept seeing this in the console - thanks cleaning this up.

Reply:
no problem, i just googled the error and it took me to the eslint docs:
https://eslint.org/docs/2.0.0/rules/array-callback-return